### PR TITLE
13.0 chbvr jco

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@yvaucher

**Comment by yvaucher** [comment]
@jco-odoo thanks for the ping, we will test that with @fclementic2c

**Comment by jco-odoo** [comment]
@oco-odoo Can you do a review on this one?

File: addons/l10n_ch/models/res_bank.py [hunk 1]
```diff
@@ -42,17 +56,24 @@ def retrieve_acc_type(self, acc_number):
         """ Overridden method enabling the recognition of swiss postal bank
         account numbers.
         """
-        if _is_l10n_ch_postal(acc_number):
+        acc_number2 = ""
+        if acc_number and " " in acc_number:
```
Comment (oco-odoo) [inline addons/l10n_ch/models/res_bank.py]: Add some comment to explain why you're doing that

File: addons/l10n_ch/models/res_bank.py [hunk 2]
```diff
@@ -42,17 +56,24 @@ def retrieve_acc_type(self, acc_number):
         """ Overridden method enabling the recognition of swiss postal bank
         account numbers.
         """
-        if _is_l10n_ch_postal(acc_number):
+        acc_number2 = ""
```
Comment (oco-odoo) [inline addons/l10n_ch/models/res_bank.py]: Better naming would be wise. "real_acc_number" "non_unique_acc_number", ... ?

**Comment by oco-odoo** [comment]
@robodoo delegate +

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@oco-odoo

**Comment by oco-odoo** [comment]
@robodoo rebase-ff

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**